packages/sources/assets/package.json

```diff
@@ -23,9 +23,9 @@
   ],
   "scripts": {
     "prebuild": "shx rm -rf dist",
-    "build": "yarn build:svg && yarn build:json",
-    "build:json": "shx rm -rf ./dist/assets/json && shx mkdir ./dist/assets/json && fantasticon",
-    "build:svg": "node src/scripts/build-svg.js && svgo -f dist/assets/svg"
+    "build": "shx mkdir -p dist/assets/svg && shx cp -r src/generated/assets/svg/all/* dist/assets/svg",
+    "prestart": "yarn build",
```

Comment:
What is this `prestart` command for ?
Is the `build` command still necessary ? Can't we just have a single `yarn start` command now ?

Reply:
No, we need both.
Start for development and build for packaging like we do for all our packages inside this monorepo.